dev/ProgressBar/ProgressBar_themeresources.xaml

```diff
@@ -1,288 +1,43 @@
-                                    <Ellipse.RenderTransform>
-                                        <TranslateTransform />
-                                    </Ellipse.RenderTransform>
-                                </Ellipse>
-                            </Border>
-                            <Border x:Name="B1" Grid.Column="8" RenderTransformOrigin="0.5,0.5">
-                                <Border.RenderTransform>
-                                    <TranslateTransform />
-                                </Border.RenderTransform>
-                                <Ellipse x:Name="E1" Fill="{TemplateBinding Foreground}" Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.EllipseDiameter}" Height="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.EllipseDiameter}" RenderTransformOrigin="0.5,0.5">
-                                    <Ellipse.RenderTransform>
-                                        <TranslateTransform />
-                                    </Ellipse.RenderTransform>
-                                </Ellipse>
-                            </Border>
-                        </StackPanel>
-                        <Border
-                            x:Name="DeterminateRoot"
-                            Background="{TemplateBinding Background}"
-                            BorderBrush="{TemplateBinding BorderBrush}"
-                            BorderThickness="{TemplateBinding BorderThickness}"
-                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                            contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
-                            <Rectangle
-                                x:Name="ProgressBarIndicator"
-                                Margin="{TemplateBinding Padding}"
-                                Fill="{TemplateBinding Foreground}"
-                                HorizontalAlignment="Left"
-                                contract7Present:RadiusX="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
-                                contract7Present:RadiusY="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}"
-                                contract7NotPresent:RadiusX="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
-                                contract7NotPresent:RadiusY="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}" />
-                        </Border>
-                    </Grid>
-                </ControlTemplate>
-            </Setter.Value>
-        </Setter>
-    </Style>
+    <x:Double x:Key="ProgressBarMinHeight">3</x:Double>
```

Comment:
Are these values correct?  How do they work with the 4px alignment grid for WinUI?

Reply:
yes, this is what design wants. 1px track + 3px indicator for default control.